### PR TITLE
Added captureCanvas method

### PR DESCRIPTION
**Pull request by olalonde**
This can be useful to call webcam.captureCanvas().toBlob for example.

### CODE REVIEW

File: src/webcam/index.js [hunk 1]
```diff
@@ -130,16 +130,24 @@ class Webcam extends Component {
...
-    const { width, height, captureFormat } = this.props;
+    const { width, height } = this.props;
 
     if (hasUserMedia && userMediaRequested) {
       const canvas = this._getCanvas();
       const ctx = canvas.getContext("2d");
 
       ctx.drawImage(this._video, 0, 0, width, height);
 
+      return canvas
```
Comment (jgradim) [inline src/webcam/index.js]: Missing `;`

**Review by jgradim — CHANGES_REQUESTED** [review]
Please remove the commit where you modify `package.json`, I have this setup like this to ensure we're using local versions of the packages, with versions defined in `package.json`

**Comment by jgradim** [comment]
Oh, and thank you for the contribution! 😄

**Comment by olalonde** [comment]
> Please remove the commit where you modify package.json

It still does that, the `./node_modules/.bin` directory is the first place npm looks for binary so it's not necessary to explicitly write it. See https://docs.npmjs.com/misc/scripts#path

Fixed the semicolon.

**Review by jgradim — APPROVED** [review]

**Comment by jgradim** [comment]
@olalonde Sorry for the delay on this! Also published `0.1.7` to npm

**Pull request merged**